include/__threading_support

```diff
@@ -93,8 +96,14 @@ typedef void* __libcpp_recursive_mutex_t[5];
 #endif
 
 // Condition Variable
+#if defined(NXDK)
+// Size and content of CONDITION_VARIABLE are different in nxdk
+typedef int32_t __libcpp_condvar_t[4];
+#define _LIBCPP_CONDVAR_INITIALIZER {0, -1, -1, 0}
```

Comment:
nit: I'd prefer to have documented that -1 is `INVALID_HANDLE_VALUE`

Reply:
I added a comment.